save-backend/src/main/kotlin/org/cqfn/save/backend/repository/FileSystemRepository.kt

```diff
@@ -101,4 +101,21 @@ class FileSystemRepository(configProperties: ConfigProperties) {
                     }
             }
     }
+
+    /**
+     * Delete a file described by [fileInfo]
+     *
+     * @param fileInfo a [FileInfo] describing a file to be deleted
+     * @return true if file has been deleted successfully, false otherwise
+     */
+    fun delete(fileInfo: FileInfo) = try {
+        Files.delete(fileInfo.getPath())
```

Comment:
why do we need to delete the file first and after that to delete a directory?

Reply:
Fixed